src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const perDaysCost = 40;
+  const discountAfterThreeDays = 20;
+  const discountAfterSevenDays = 50;
+
+  if (days < 3) {
+    return days * perDaysCost;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
```suggestion
  if (days < 7) {
```
It's not necessary here